### PR TITLE
append UTC to timestamp

### PR DESCRIPTION
**Pull request by cptnuss-ops**
Append "UTC" to the timestamp, so that the discord user gets informed about the display offset.
The discord module only has a function to return UTC, a simple way to get the session offset and display the time from the actual discord timezone will be investigated later.

### CODE REVIEW

**Comment by k8thekat** [comment]
My only request is to change your formatters line width from 80 to something over 150..  Especially if its black; it tends to word wrap every line and makes it hard to follow.

Thanks.

**Comment by cptnuss-ops** [comment]
Should be better now :)

**Review by k8thekat — APPROVED** [review]
